### PR TITLE
feat: add backend for getting work flows from data index service

### PR DESCRIPTION
**Pull request by rhkp**
Hi @caponetto 
Please approve or provide feedback of this PR, which is worked upon after your upstream updates. 

Thanks.

### CODE REVIEW

File: plugins/3scale-backend/dist-dynamic/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@janus-idp/backstage-plugin-3scale-backend-dynamic",
-  "version": "1.1.2",
+  "version": "1.3.0",
```
Comment (caponetto) [inline plugins/3scale-backend/dist-dynamic/package.json:3]: We should not touch other plugins, only `orchestrator*`. Please revert the changes on them.

Comment (rhkp) [reply plugins/3scale-backend/dist-dynamic/package.json:3]: Okay sure. I think yesterday while try different `yarn install` commands they got auto changed.

File: plugins/orchestrator-backend/src/types/apiResponse.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+export interface ApiResponse {
```
Comment (tiagodolphine) [inline plugins/orchestrator-backend/src/types/apiResponse.ts:1]: We may review this later, but it should follow a pattern in all the orchestrator backend APIs, for example, here the return payload contains(result, message), but this is not what other APIs follow, at some point we need to standardize all APIs, this is probably related to the task Gloria is working on with the OpenApi for the backend plugin.

**Review by tiagodolphine — COMMENTED** [review]
LGTM, I just added a comment related the the return format of the REST API, and the other thing is related to the versions that were updated in dependencies on other plugins that should be reverted.

File: plugins/orchestrator-backend/src/types/apiResponse.ts (hunk 2 above)
Comment (rhkp) [reply plugins/orchestrator-backend/src/types/apiResponse.ts:1]: sure, noted.

**Comment by rhkp** [comment]
Hi @caponetto / @tiagodolphine,
Are there any more comments or feedback here to take care of? 

If not, can we please merge this one? 
Thanks.

File: plugins/orchestrator-backend/README.md [hunk 3]
```diff
@@ -3,3 +3,42 @@
...
+```shell
+export DATA_INDEX_URL=http://<server:port>/graphql/
+```
+
+### Getting workflow list
+
+- Test the plugin to see if it returns deployed workflows.
+
+```shell
+curl localhost:7007/api/orchestrator/dataIndexWorkflows | jq .
```
Comment (caponetto) [inline plugins/orchestrator-backend/README.md:24]:
```suggestion
curl localhost:7007/api/orchestrator/workflows/definitions | jq .
```
since the endpoint changed during the review.

**Review by tiagodolphine — APPROVED** [review]

**Review by caponetto — APPROVED** [review]
